test/cmdlineTests/yul_unimplemented/err

```diff
@@ -1,5 +0,0 @@
-Error (1834): Unimplemented feature error in <FILENAME REMOVED>
```

Comment:
This test is now useless (it is called `yul_unimplemented`). Can you come up with something new? Or if not, then just delete it.

Reply:
I wanted to wait for the rest of my Sol->Yul PRs to be merged to come up with something for this test... Will try maybe before merging them if I fond something.